### PR TITLE
COAMPS-TC support

### LINKED ISSUE #850: Add support for COAMPS-TC

### PR DESCRIPTION
**Pull request by jasonfleming**
resolves #850

### CODE REVIEW

File: downloadCOAMPSTC.sh [hunk 1]
```diff
@@ -0,0 +1,490 @@
+#!/bin/bash
```
Comment (wwlwpd) [inline downloadCOAMPSTC.sh:1]: We need to start putting all scripts in `./bin` or somewhere else that's in `PATH`. If it's not in `PATH`, then we need to add it there. `$SCRIPTDIR` is not currently in `PATH` but probably should be just for the existing scripts there.

**Review by wwlwpd — CHANGES_REQUESTED** [review]
Big request atm: pls put `downloadCOAMPSTC.sh` into `./bin` or some other directory that's in `PATH`. If you need to create a new `bin` somewhere and it should be in `PATH`, this is controlled in `./cloud/general/asgs-brew.pl`.

File: downloadCOAMPSTC.sh (hunk 1 above)
Comment (jasonfleming) [reply downloadCOAMPSTC.sh:1]: can do

**Comment by jasonfleming** [comment]
Hey @wwlwpd before I move `downloadCOAMPSTC.sh` to `/bin`, I wanted to point out that this script is not meant to be executed as a standalone, it is meant to be `source`d into `asgs_main.sh` ... I didn't know if this changes your mind about it

**Pull request closed without merging**